storage/path.go

```diff
@@ -24,14 +24,14 @@ func BlockDevicePath(device BlockDevice) (string, error) {
 		return diskByWWN + device.WWN, nil
 	}
 	if device.HardwareId != "" {
-		return filepath.Join(diskByID, device.HardwareId), nil
+		return path.Join(diskByID, device.HardwareId), nil
 	}
 	if len(device.DeviceLinks) > 0 {
 		// return the first device link in the list
 		return device.DeviceLinks[0], nil
 	}
 	if device.DeviceName != "" {
-		return filepath.Join(diskByDeviceName, device.DeviceName), nil
+		return path.Join(diskByDeviceName, device.DeviceName), nil
```

Comment:
Right. This is specifically one of those cases where you aren't talking about a local path, but a path on some other machine. We'll need to be careful about how we do them, but filepath is definitely not the answer.